can-define.js

```diff
@@ -288,6 +288,7 @@ make = {
 		events: function(prop, getCurrent, setData, eventType) {
 			return function(newVal) {
 				if (this.__inSetup) {
+					Observation.ignore(getCurrent).call(this);
```

Comment:
In the cases we're looking at, `getCurrent()` is a callback created by `make.read.data()`, which itself doesn't have an __inSetup check.  None of the `make.read` getters call `Observation.add` directly but I don't know what happens with, say, a compute.